static/templates/settings/account-settings.handlebars

```diff
@@ -80,3 +83,8 @@
 
   </div>
 </div>
+<script>
+$(function () {
+    $('[data-toggle="tooltip"]').tooltip();
+});
+</script>
```

Comment:
@timabbott Thanks for reviewing my code. I've made the changes as per your suggestions